watchgod/main.py

```diff
@@ -1,15 +1,15 @@
-import asyncio
 import functools
 import logging
 import os
 import signal
-from concurrent.futures import ThreadPoolExecutor
 from functools import partial
 from multiprocessing import Process
 from pathlib import Path
 from time import time
 from typing import TYPE_CHECKING, Any, Awaitable, Callable, Dict, Generator, Optional, Set, Tuple, Type, Union, cast
 
+import anyio  # type: ignore
```

Comment:
(As a separate PR) it might be worth adding a `.mypy.ini` file and configuring this there. Also, generally worth adding examples and tests to the mypy run. e.g. See the aio-libs configs: https://github.com/aio-libs/aiohttp-devtools/blob/master/.mypy.ini

Reply:
You can configure mypy fully in setup.cfg or pyproject.toml, no need for another file.

Reply:
Sure, if you prefer. I just find our configs a little too big to be comfortable squeezing in with everything else.

Reply:
Sorry, I can't help try to reduce the number of files in the root of the repo.

Reply:
The actual problem here was that anyio wasn't being installed during mypy tests.